app/pages/lab-fem/workflow.jsx

```diff
@@ -506,6 +514,29 @@ class EditWorkflowPage extends Component {
 
             <hr />
 
+            {Array.from(this.props.project.experimental_tools).includes('caesarEnabled') ?
+              <div>
+                <p>
+                  <label>
+                    <input
+                      type='checkbox'
+                      checked={isCaesarEnabled}
+                      onChange={this.toggleCaesarEnabled}
+                    />
+                    Caesar Enabled
```

Comment:
```suggestion
                    Caesar Data Fetching Enabled
```

Reply:
Revised text makes clear this is related to data fetching, not related to other Caesar use cases (early retirement, etc).